test/required_files/agent-deb.txt

```diff
@@ -0,0 +1,14 @@
+/lib/systemd/system/datadog-agent-security.service
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Challenging if `test` is the proper location for these files. These look rather like fixtures to the packaging jobs, so maybe they should belong there? No strong opinion though